python/cuml/common/input_utils.py

```diff
@@ -195,7 +211,7 @@ def input_to_cuml_array(X, order='F', deepcopy=False,
             warnings.warn("Expected " + order_to_str(order) + " major order, "
```

Comment:
What about changing these warnings to debug level so they stay around but are less prominent?